src/ATen/nn_parse.py

```diff
@@ -86,6 +88,8 @@ def get_return(args):
     'weights': 'weight',
     'train': 'training',
     'val': 'value',
+    'lambda': 'lambd',
```

Comment:
This is because of python keywords, right?

Reply:
Yeah